src/components/kwc-auth-login.ts

```diff
@@ -55,45 +77,53 @@ export class Login extends LitElement {
      /**
     * Returns the current value of the eye image field or an empty string
     */
-
     get eyeimage() {
         return this.imageInput ? this.imageInput.value : '';
     }
 
+    constructor() {
+        super();
+        this.loginGliph = '';
+    }
 
     render() {
         return html`
          ${templateContent(button)}
-        <div id="login">
-            <div class="login-section">     
-                <form class="form-wrapper" @submit=${this._submit}>
-                    <h2 class="login-title">Login to your account</h2>
+        <div>
+            <div class="header"
+                style="background-image:url(${this.loginGliph})">
+                <h2>Login</h2>
+            </div>
+            <form @submit=${this._submit}>
+                <div class="input-wrapper">
+                    <label for="username">Username</label>
+                    <input type="text" id="username" placeholder="Your Kano Username"/>
+                    <label for="password">Password</label>
                     <div class="input-wrapper">
-                        <label for="username">Username</label>
-                        <input class="input" type="text" id="username" placeholder="Your Kano Username"/>
-                        <label for="password">Password</label>
-                        <div class="input-password-wrapper">
-                            <input class="input" type="password" id="password" placeholder="Make up a secret password"/>
-                            <img src="https://imgplaceholder.com/42x32/transparent/757575/fa-eye-slash" class="eye-toggle" id="eyeimage" @click="${this.togglePassword}"/>
-                        </div>
+                        <input type="password" id="password" placeholder="Make up a secret password"/>
+                        <img src="https://imgplaceholder.com/42x32/transparent/757575/fa-eye-slash" class="eye-toggle" id="eyeimage" @click="${this.togglePassword}"/>
```

Comment:
As soon as we get svgs we should create a `icons.js` and embed it there instead of this

Reply:
Also, some browser already add this by default it might overlap and look weird in some cases

Reply:
Yeah, I was just working with Natasha's implementation. Not too sure about it. But SVGs for sure